### DISCUSSION TITLE
how to use account not upn authentication

### DISCUSSION THREAD

**Discussion by HanJoey** [neos-sdi/adfsmfa#318] [Show and tell]
how to use account not upn authentication

**Comment by redhook62** [comment]
In identity federation scenarios, it is recommended to use UPN.

However, when installing the plugin you can choose to use the Windows Account Name as indicated in the documentation here: [Product Activation](https://github.com/neos-sdi/adfsmfa/wiki/01-Installation#activating-product) .

All accounts already registered must be deleted and must register again after the change of identity claim.
